src/test/resources/view/UiPartTest/validFileWithFxRoot.fxml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<fx:root type="seedu.address.ui.TestFxmlObject" xmlns="http://javafx.com/javafx/17"
-        xmlns:fx="http://javafx.com/fxml/1">
+<?import keycontacts.ui.TestFxmlObject?>
```

Comment:
This import should be unnecessary. Doesn't really matter so up to you if you want to remove it

Reply:
See above